src/index.js

```diff
@@ -1,29 +1,66 @@
+// @ts-check
 import analyticModules from './modules';
 
 const globalProperties = {};
 
 export const Analytics = {
+  /**
+   * @param {import("./main").IInitializeParams} options
+   */
+  initialize(options) {
```

Comment:
@AlanLes please describe this part in README.md, how to use it etc.

Reply:
It's done ;)